src/Process/3dTilesProcessing.js

```diff
@@ -254,6 +287,8 @@ export function init3dTilesLayer(view, scheduler, layer) {
                 tile.updateMatrixWorld();
                 layer.tileIndex.index[tile.tileId].loaded = true;
                 layer.root = tile;
+                layer.extent = boundingVolumeToExtent(layer.projection || view.referenceCrs,
+                    tile, tile.boundingVolume, tile.matrixWorld);
```

Comment:
`tile` should have been `tile.layer` (but this arguments is unused and should go away)